extensions/Prefixes.md

```diff
@@ -30,6 +30,7 @@ Once a prefix is approved, it should be added to the [glTF Validator's list of k
 | `AVR` | AltspaceVR, Inc. | https://altvr.com/ | [#1009](https://github.com/KhronosGroup/glTF/issues/1009) |
 | `BENTLEY` | Bentley Systems, Inc. | https://www.bentley.com/<br>`paul.connelly at bentley.com` | [#2259](https://github.com/KhronosGroup/glTF/issues/2259) |
 | `BLENDER` | Blender Foundation | https://www.blender.org/ | [#865](https://github.com/KhronosGroup/glTF/issues/865) |
+| `C3D` | Core3D, Inc. | https://core3d.io/ | [#2264](https://github.com/KhronosGroup/glTF/issues/2264) |
```

Comment:
Based on the HTTP response (301), the canonical URI should be `https://www.core3d.io/`.

Reply:
I can update that but ...

1. does it matter?
2. what about the ones that, from a **very** quick test, seem to respond similarly...?

```
https://adobe.com/  : HTTP/1.1 301 Moved Permanently
https://agi.com/  : HTTP/1.1 301 Moved Permanently
http://animech.com/  : HTTP/1.1 301 Moved Permanently
https://audaces.com/ : HTTP/1.1 301 Moved Permanently
https://core3d.io/  : HTTP/1.1 301 Moved Permanently
https://www.epicgames.com/  : HTTP/1.1 301 Moved Permanently
https://foxitsoftware.com/ : HTTP/1.1 301 Moved Permanently
https://www.hevolus.it/  : HTTP/1.1 301 Moved Permanently
https://www.intel.com/  : HTTP/1.1 301 Moved Permanently
https://matterport.com/  : HTTP/1.1 302 Moved Temporarily
https://www.nvidia.com/  : HTTP/1.1 307 Temporary Redirect
https://github.com/omigroup/OMI  : HTTP/1.1 301 Moved Permanently
https://otoy.com/  : HTTP/1.1 301 Moved Permanently
https://www.se.com/  : HTTP/1.1 302 Moved Temporarily
https://spectrumcustomizer.com/  : HTTP/1.1 301 Moved Permanently
```

Reply:
> does it matter?

If a redirect gets removed or disabled for some reason, the link may no longer work. This is a rare case, though.

> what about the ones ...

Most links in the table were copied as-is without checking.

There could be a separate PR updating URIs with 301.